src/ipvs/ip_vs_dest.c

```diff
@@ -63,8 +74,12 @@ static void __dp_vs_dest_update(struct dp_vs_service *svc,
 
     if (udest->max_conn == 0 || udest->max_conn > dest->max_conn)
         dest->flags &= ~DPVS_DEST_F_OVERLOAD;
-    dest->max_conn = udest->max_conn;
-    dest->min_conn = udest->min_conn;
+    dest->max_conn = udest->max_conn / num_lcores;
+    dest->min_conn = udest->min_conn / num_lcores;
+    if (rte_lcore_id() == first_lcore_cid) {
+        dest->max_conn += udest->max_conn % num_lcores;
+        dest->min_conn += udest->min_conn % num_lcores;
+    }
```

Comment:
I think exact max/min_conn don't make sense because the conns on different workers are just coarsely the same.
Considering the worker number is much smaller the conns, maybe something like the following is enough.

```
dest->max_conn = (udest->max_conn+num_lcores) / num_lcores;
dest->min_conn = (udest->min_conn+num_lcores)  / num_lcores;
```

Reply:
Yes, max_conn is generally much larger than the number of lcore. 
If we use dest->max_conn = (udest->max_conn+num_lcores -1) / num_lcores;
the problem is that the max_conn displayed through ipvsadm -ln --thresholds is not very friendly, 
because the displayed max_conn is not consistent with the actual configured max_conn.

Reply:
We don't set conn limits in most cases, where the `udest->max_conn` and `udest->min_conn` are zero.
In consideration of performance,  we can spare the efforts of calculating per-lcore's conn limit values when conn limits are not specified.

Reply:
I have removed the loop for getting the first forwarding lcore, put the excess conn to the rs in the master lcore, to ensure that the sum of max conn is consistent with the configured max conn.